cyhy/mailer/cli.py

```diff
@@ -796,6 +797,7 @@ def send_cyhy_reports(
         for request in cyhy_requests:
             id = request["_id"]
             acronym = request["agency"]["acronym"]
+            enrolled = request["enrolled"]
```

Comment:
There are a handful of entities that don't have enrolled dates, so just to be safe, we should change this to:
```suggestion
            enrolled = request.get("enrolled", "n/a")
```

If you have a preference for something other than "n/a" there, feel free to adjust it.